src/docfx/build/legacy/metadata/LegacyMetadata.cs

```diff
@@ -66,9 +66,10 @@ public static JObject GenerateLegacyRedirectionRawMetadata(Docset docset, PageMo
             rawMetadata = GenerataCommonMetadata(rawMetadata, docset);
             rawMetadata["conceptual"] = content;
 
-            var fileRelativePath = PathUtility.NormalizeFile(Path.GetRelativePath(file.Docset.Config.SiteBasePath, file.OutputPath));
+            var path = PathUtility.NormalizeFile(Path.GetRelativePath(file.Docset.Config.SiteBasePath, file.SitePath));
 
-            rawMetadata["_path"] = rawMetadata["fileRelativePath"] = fileRelativePath;
+            rawMetadata["_path"] = path;
+            rawMetadata["fileRelativePath"] = Path.ChangeExtension(path, ".html");
```

Comment:
what's that used for, do you need locale and moniker info? and index site path doesn't have extension

Reply:
https://ceapex.visualstudio.com/Engineering/All/_git/Docs.DocFX.Impact/pullrequest/4237?_a=files&path=%2Fsource%2Fazure-docs-pr.output%2Fazure%2Factive-directory%2Fauthentication%2Factive-directory-passwords-faq.raw.page.json